packages/clarity-js/src/interaction/encode.ts

```diff
@@ -117,13 +117,17 @@ export default async function (type: Event, ts: number = null): Promise<void> {
         case Event.Scroll:
             for (let entry of scroll.state) {
                 let sTarget = metadata(entry.data.target as Node, entry.event);
+                const top = metadata(entry.data.top as Node, entry.event);
+                const bottom = metadata(entry.data.bottom as Node, entry.event);
+                const cTopHash = top.hash ? top.hash.join(Constant.Dot) : Constant.Empty;
```

Comment:
>cTopHash [](http://example.com/codeflow?start=22&length=8)

sTopHash / sBottomHash right? The s stands for scroll and I think you grabbed these from the click section